recipes/srreal/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "srreal" %}
+{% set version = "v1.2" %}
+{% set sha256 = "d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/diffpy/{{ name }}/archive/{{ version }}.tar.gz
```

Comment:
Might be better to use a sdist tarball from pypi.io
https://pypi.org/project/diffpy.srreal/#files